docs/specs/ops.yml

```diff
@@ -267,3 +267,18 @@ repos:
   - files:
       ContentTemplate/schemas/TestData.schema.json: "{}"
 outputs:
+---
+# SourceInfo of incorrect opConfig
+repos:
+  https://github.com/redirection/SourceInfoOfIncorrectOpConfig:
+  - files:
+      .openpublishing.publish.config.json: |
+        {
+          "docsets_to_publish": [{"build_source_folder":"a"}],
+          "redirection_files": "a/b/.openpublishing.redirection.test.json",
+        }
+      a/docfx.yml:
+outputs:
+  .errors.log: |
+    {"message_severity":"error","code":"violate-schema","message":"Error converting value \"a/b/.openpublishing.redirection.test.json\" to type 'System.Collections.Generic.HashSet`1[Microsoft.Docs.Build.PathString]'.","file":".openpublishing.publish.config.json","line":3,"column":66}
+    {"message_severity":"error","code":"violate-schema","message":"Error converting value \"a/b/.openpublishing.redirection.test.json\" to type 'System.Collections.Generic.HashSet`1[Microsoft.Docs.Build.PathString]'.","file":".openpublishing.publish.config.json","line":3,"column":66,}
```

Comment:
>, [](start = 283, length = 1)

extra `,`

Besides, would these errors be deduped by OP report?

Reply:
Won't dedup errors in this PR.